app/controllers/api/v1/ontologies_controller.rb

```diff
@@ -1,14 +1,25 @@
-# Base controller for all API controllers
 class Api::V1::OntologiesController < Api::V1::Base
-
   inherit_resources
-  actions :index, :update
-  before_filter :check_write_permission, :except => [:index, :show]
-  has_scope :repository_id, :basepath, :without_parent
+  actions :show
+
+  respond_to :text, only: %i(show)
 
-  def index
-    collection
-    render :index
+  def show
+    super do |format|
+      format.json { render json: resource, serializer: OntologySerializer }
+      format.text { send_download }
+    end
   end
 
+  private
+  def send_download
+    asset = version || resource
+    render text: asset.file_in_repository.content,
+           content_type: Mime::Type.lookup('application/force-download')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.